content/posts/February-2022.md

```diff
@@ -12,3 +12,8 @@ weight: 64
 - Packit-service now supports re-triggering against failed task using comment on
   pull request `/packit rebuild-failed` and similarly for testing farm: `/packit retest-failed`.
   ([packit-service#1303](https://github.com/packit/packit-service/pull/1303))
+
+## Week 6 (February 8th - February 14th)
+
+- Packit now correctly handles colons in git trailer values in source-git commits.
+  ([packit#1478](https://github.com/packit/packit/pull/1478))
```

Comment:
how about F36 and that we correctly support it right now? is it worth mentioning?

Reply:
I thought about that at first but don't think it's relevant to users. (And there was no changelog entry in the PR as well.) We basically haven't changed any behaviour.

Reply:
From code PoV you're correct. But the fact that we'll restart pods and purge the cache will pick up F36 :)

I'm still LGTM, merge at your will.